### DISCUSSION TITLE
Trial Chambers, Misc. Blocks n Items

### DISCUSSION THREAD

**Discussion by Bryce-Knight** [Roadhog360/Et-Futurum-Requiem#522] [Ideas]
I see the note about no blocks and items but there's an important block missing in my opinion: The Grindstone. I need the decoration.

Now for the important part Trial Chambers. It could be configurable to set the biomes it can spawn in and its frequency or just to turn them off entirely. The new blocks that it adds, the breeze mob would be awesome and funny. I wouldn't go as far as to say add the Vaults and Keys same with the Ominous Potion. The Copper Cauldron/Sink from a previous joke version that can be found in the Chambers.

ALSO GOATS WITH CONFIGURABLE KNOCKBACK (I have evil plans)

Also a question to go with this and I'm sorry if you've been asked this a lot but how's water logging backport mod looking and what all would it bring with it?

EDIT: Moss Blocks and the blocks from the lush caves please 🙏

**Comment by Roadhog360** [comment]
There currently has been no progress on any fluid logging mod for 1.7.10. Most of the trial chamber blocks are in as of 2.6.0 but no structure unfortunately.

**Reply by Bryce-Knight** [reply]
awwww well thats better than nothing thanks for the heads up! and I cant wait till the fluid mod is available. I probably wont leave 1.7.10 when that happens honestly

**Reply by Roadhog360** [reply]
I would say having structures isn't too far off at this point, though. I already have a fully working library that I made, which can read the modern NBT structure data in 1.7.10. The only thing I'm missing is just to actually insert them into the game as real structures. But modern structures are not too far off. I love the new trial chambers too and hope to make them configurable to be the perfect fit for a modpack of any theme!

**Reply by Bryce-Knight** [reply]
OOO WAIT hear me out what if there were different variants of them that matches the corresponding mods? Like new blocks to breathe new life into old mods but make it still related to the structure. Like DBC, Witchery, Draconic Evo., TINKERS I HAVE SO MANY IDEAS NOW sorry but

I'd 1000% love that and would love to help somehow
